src/test/compile-fail/issue-18252.rs

```diff
@@ -14,6 +14,5 @@ enum Foo {
 
 fn main() {
     let f = Foo::Variant(42);
-    //~^ ERROR uses it like a function
-    //~| struct called like a function
+    //~^ ERROR expected function, found struct variant `Foo::Variant`
```

Comment:
This seems to indicate we've removed the text on the primary error.  I'd like to keep labels.  This one specifically I think is helpful.